packages/editor/src/components/editors/NameEditor.tsx

```diff
@@ -3,8 +3,8 @@ import { memo } from 'react';
 import { useNamePart } from '../parts';
 import InscriptionEditor from './InscriptionEditor';
 
-const NameEditor = memo((props: { icon: IvyIcons; hideTags?: boolean }) => {
-  const name = useNamePart({ hideTags: props.hideTags });
+const NameEditor = memo((props: { icon: IvyIcons; hideTags?: boolean; disableName?: boolean }) => {
+  const name = useNamePart({ hideTags: props.hideTags, disableName: props.disableName });
```

Comment:
I think this change here is not needed, as noone uses this NameEditor with the disabledName right? This is only the case for the process editor and those don't use the NameEditor.